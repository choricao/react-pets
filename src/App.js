import React, { Component } from 'react';
import Pet from './components/Pet';
import './App.css';

const PET_LIST = [
  {
    name: 'Bogart',
    age: 8,
    breed: 'Mixed',
    about: 'Bogart was Kirsten\'s Great cat!',
  },
  {
    name: 'Atticus',
    age: 2003,
    breed: 'Irish Wolfhound',
    about: 'Atticus is a senior citizen!',
  },
  {
    name: 'Bagel',
    age: 1,
    breed: 'Beagle',
    about: 'Bagel the Beagle',
  }
];

class App extends Component {
  render() {
    return (
      <div className="App">
        <Pet
          name={PET_LIST[0].name}
          age={PET_LIST[0].age}
          breed={PET_LIST[0].breed}
          about={PET_LIST[0].about}
          />
      </div>
    );
  }
}

export default App;
