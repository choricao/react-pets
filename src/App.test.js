import React from 'react';
import ReactDOM from 'react-dom';
import { mount, shallow } from 'enzyme';
import App from './App';

describe('<App />', () => {

  // test('will match the last snapshot with deep rendering', () => {
  //   const wrapper = mount(<App />);
  //   expect(wrapper).toMatchSnapshot();
  //   wrapper.unmount();
  // });

  test('that it renders App with shallow rendering', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
