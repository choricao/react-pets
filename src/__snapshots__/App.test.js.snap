// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> that it renders App with shallow rendering 1`] = `
<div
  className="App"
>
  <PetCollection />
</div>
`;
